examples/vision/md/siamese_network.md

```diff
@@ -83,10 +84,20 @@ positive_images_path = cache_dir / "right"
 
 <div class="k-default-codeblock">
 ```
-zsh:1: command not found: gdown
-zsh:1: command not found: gdown
-unzip:  cannot find or open left.zip, left.zip.zip or left.zip.ZIP.
-unzip:  cannot find or open right.zip, right.zip.zip or right.zip.ZIP.
+/home/scottzhu/.local/lib/python3.10/site-packages/gdown/cli.py:126: FutureWarning: Option `--id` was deprecated in version 4.3.1 and will be removed in 5.0. You don't need to pass it anymore to use a file ID.
```

Comment:
Please fix (you can just update the code in all 3 files and scrub the warning by hand if you don't want to regenerate)

Reply:
Done. remove the warning.